scss/_patterns_search-and-filter.scss

```diff
@@ -0,0 +1,183 @@
+@import 'settings';
+
+@mixin vf-p-search-and-filter {
+  @include vf-search-and-filter;
+  @include vf-filter-panel-section;
+}
+
+@mixin vf-search-and-filter {
+  .p-search-and-filter {
+    $height: 2.5rem;
+
+    border: 1px solid $color-mid-dark;
+    position: relative;
+
+    .p-search-and-filter__search-container {
+      align-items: center;
+      background-color: $color-x-light;
+      display: flex;
+      flex-wrap: wrap;
+      height: auto;
+      margin: 0;
+      min-height: $height;
+      overflow: hidden;
+      padding: 0 2rem 0 0.5rem;
+      position: relative;
+
+      &[data-active='true'] {
+        height: auto;
+      }
+
+      &[data-empty='false'],
+      &[aria-expanded='false'] {
+        height: $height;
+      }
+
+      .p-chip {
+        margin-bottom: 0.5rem;
+      }
+
+      .p-search-and-filter__selected-count {
+        color: $color-information;
+        cursor: pointer;
+        position: absolute;
+        right: 0.5rem;
+        top: 6px;
+      }
+
+      &[aria-expanded='false'] {
+        .p-chip {
+          margin-bottom: 0.5rem;
+        }
+      }
+
+      &[aria-expanded='true'] {
+        height: auto;
+        max-height: 100%;
+
+        .p-search-and-filter__selected-count {
+          display: none;
+        }
+      }
+
+      .p-search-box {
+        flex-grow: 1;
+      }
+    }
+
+    .p-icon--search {
```

Comment:
Why do we have this icon in the markup at all if we are just hiding it with `display: none`?

Reply:
I think your right, I went back and forward on that a few times.